slides/04-evaluating-models.qmd

```diff
@@ -134,17 +300,17 @@ tree_fit %>%
 ::: columns
 ::: {.column width="50%"}
 ```{r}
-tree_fit %>%
-  augment(frog_train) %>%
-  rmse(latency, .pred)
+taxi_fit %>%
+  augment(taxi_train) %>%
+  accuracy(truth = tip, estimate = .pred_class)
 ```
 :::
 
 ::: {.column width="50%"}
 ```{r}
-tree_fit %>%
-  augment(frog_test) %>%
-  rmse(latency, .pred)
+taxi_fit %>%
+  augment(taxi_test) %>%
+  accuracy(truth = tip, estimate = .pred_class)
```

Comment:
Could we pass these arguments (and the same ones in the LHS column) by position so that the passed column name isn't clipped in the output? Or maybe put these on top of each other instead of in columns?

Reply:
Done!